modules/op-tkn-condition-management.adoc

```diff
@@ -21,6 +21,14 @@ Delete a Condition.
 $ tkn condition delete mycondition1 -n myspace
 ----
 
+== condition describe
+Describe a Condition.
+
+.Example: Describe the `mycondition1` Condition in a namespace
```

Comment:
It looks like this is consistent with how you use it throughout this doc, but I'd consider saying "in the `myspace` namespace", throughout the examples in the doc. Since you specifically call out myCondition1 and say "in a namespace" in the title but don't use angle brackets around <myspace> to indicate that it's replaceable.

Reply:
Hello @bergerhoffer - This was debated before as well, QE was of the opinion of removing the "in the `myspace` namespace" altogether from the example. As per them it sounded repetitive and did not add any value. We concluded to remove just  "`myspace`"and retain only "in a namespace".